api/src/main/java/org/pmiops/workbench/opsgenie/EgressEventServiceImpl.java

```diff
@@ -43,6 +46,12 @@
   private final Provider<WorkbenchConfig> workbenchConfigProvider;
   private final UserService userService;
   private final WorkspaceAdminService workspaceAdminService;
+  private final WorkspaceDao workspaceDao;
+
+  // Workspace namespace placeholder in Egress alert when workspace namespace is missing.
+  // It may happens when workspace is deleted from db, or we are not able to retrieve workspace
+  // by google project id.
+  @VisibleForTesting static final String NOT_FOUND_WORKSPACE_NAMESPACE = "NOT FOUND";
```

Comment:
nit: maybe NOT_FOUND instead; this will be inlined into some URLs in the workspace alert so it may look odd with the space